test/unit/test_ArithmeticExpressions.cpp

```diff
@@ -4,6 +4,7 @@
 
 #include <gtest/gtest.h>
 #include "LRALogic.h"
+#include "FastRational.h"
```

Comment:
Adding this include is not necessary after the new tests has been moved to a different file.

Reply:
removed.